### ISSUE TITLE
[seven-guis-sveltekit] 7. Create 7 guis dropdown menu

### ISSUE THREAD

**Issue by kenslachtajr** [thisdot/starter.dev-github-showcases#1507]
# Description
The purpose of this ticket is to create the drop down menu which will hold all of the 7 GUIs. Selecting an item will navigate to that page. If those pages do not exist, that should be created so that they can be linked to the menu items for navigation

# Acceptance Criteria
- [ ] The menu exists and includes all seven of the GUI items in the screenshot
- [ ] Selecting an item in the menu navigates to the selected page

![Image](https://user-images.githubusercontent.com/1828602/221106284-f5549526-4af2-43f5-af40-e95691c3f577.png)

**Comment by kodejuice** [comment]
This seems to be linked to [[seven-guis-sveltekit] 6. Create navigation header](https://github.com/thisdot/starter.dev-7guis-showcases/issues/6)

**Comment by kenslachtajr** [comment]
Yes, https://github.com/thisdot/starter.dev-7guis-showcases/issues/6 needs to be done before this ticket can be completed.

**Comment by dustinsgoodman** [comment]
Created in the wrong repo. Closing.
